### PR TITLE
Sync from 1cafcb39c8a101893ae30f92a0a789e2c1d8b85b (3.1.0)

### PR DESCRIPTION
**Pull request by kakawait**
- [x] SCSS (except FontAwesome stuffs)
- [x] JS
- [x] i18n files
  - @dojineko if you have time to review `ja.yaml` (I ping you because you already contribute on that file). Thank you
  - @italocegatta if you have time to review `pt-br.yaml` (I ping you because you already contribute on that file). Thank you
  - @mbravorus if you have time to review `ru.yaml` (I ping you because you already contribute on that file). Thank you
  - @flojon if you have time to review `sv-se.yaml` (I ping you because you already contribute on that file). Thank you
  - @dominhhai if you have time to review `vi.yaml` (I ping you because you already contribute on that file). Thank you
- [x] HTML
- [x] gitalk
- [x] FontAwesome 5
- [x] Docs

--- 

- [x] rebuild prod

### CODE REVIEW

**Comment by dominhhai** [comment]
LGTM

File: i18n/sv-se.yaml [hunk 1]
```diff
@@ -115,6 +130,26 @@
...
+- id: "post.share"
+  translation: "Dela det här inlägget"
+
+- id: "post.gallery"
+  translation: "Galleri"
+
+## POST.IMAGE ##
+
+- id: "post.image.one"
+  translation: "image"
```
Comment (flojon) [inline i18n/sv-se.yaml]: Swedish translation should be:
"bild"

File: i18n/sv-se.yaml [hunk 2]
```diff
@@ -115,6 +130,26 @@
...
+
+- id: "post.image.other"
+  translation: "{n} images"
```
Comment (flojon) [inline i18n/sv-se.yaml]: Swedish translation should be:
"{n} bilder"

**Comment by flojon** [comment]
Two strings are missing in swedish otherwise it looks good to me

**Comment by kakawait** [comment]
@flojon thank you so much for review. I'm currently switching my main laptop (and moving from macos to linux) so need to configure it and I'll continue that PR

**Pull request merged**